src/theme/system.ts

```diff
@@ -0,0 +1,80 @@
+// ANCHOR
+import { radii } from './sizes.theme';
+
+// VENDOR
+import {
+    get,
+    system,
+    TLengthStyledSystem,
+    ResponsiveValue,
+} from 'styled-system';
+import * as CSS from 'csstype';
+
+export {
+    compose,
+    color,
+    ColorProps,
+    background,
+    BackgroundProps,
+    padding,
+    PaddingProps,
+    margin,
+    MarginProps,
+    display,
+    DisplayProps,
+} from 'styled-system';
+
+export const variant = <T extends {}>({
```

Comment:
Is this still vendor code all the way down to lines 43?

Reply:
Just the imports